app/src/main/java/buggy/plandroid/com/plandroidbugs/UserAdapter.java

```diff
@@ -1,27 +1,78 @@
 package buggy.plandroid.com.plandroidbugs;
 
 
+import android.app.Dialog;
+import android.content.Context;
 import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.TextView;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
+import buggy.plandroid.com.plandroidbugs.app.PlanDroidApp;
 import buggy.plandroid.com.plandroidbugs.room.UserEntity;
 
 public class UserAdapter extends RecyclerView.Adapter<ViewHolder> {
+
+    private static final String COLLABORATOR_ROLE = "17dce2c5-4931-47e7-8c98-84b35f00ba03";
+    private static final String POWER_COLLABORATOR_ROLE = "e7295fe5-5312-4559-b784-a74498464fb7";
+    public static final String ADMIN_ROLE ="9d139e64-cac9-4f23-b4d5-9fd3688b498e";
+
```

Comment:
Try to put this into another class, we don't like to have this kind of validations inside an adapter that should worry only about inflating info into a list.